.mailmap

```diff
@@ -374,7 +374,8 @@ Ken Cochrane <kencochrane@gmail.com> <KenCochrane@gmail.com>
 Ken Herner <kherner@progress.com> <chosenken@gmail.com>
 Ken Reese <krrgithub@gmail.com>
 Kenfe-Mickaël Laventure <mickael.laventure@gmail.com>
-Kevin Alvarez <crazy-max@users.noreply.github.com>
+Kevin Alvarez <github@crazymax.dev>
+Kevin Alvarez <github@crazymax.dev> <crazy-max@users.noreply.github.com>
```

Comment:
Can you put the one missing: https://github.com/moby/buildkit/blob/f345540efabe1582a365d172f72f50594b822a06/.mailmap#L17-L19

Thanks!